firmware/console/binary/tunerstudio.h

```diff
@@ -42,8 +42,6 @@ bool handlePlainCommand(TsChannelBase* tsChannel, uint8_t command);
  */
 void handleQueryCommand(TsChannelBase* tsChannel, ts_response_format_e mode);
 
-char *getWorkingPageAddr();
```

Comment:
if it's not in header shall we make it static to emphasize?

Reply:
fixed